### PR TITLE
Setup packer

### PR DESCRIPTION
**Pull request by zaro0508**
Setup packer files to build and deploy AWS AMIs.
Create an initial ubuntu weechat AMI.

to build and deploy image:
```
cd packer/weechat
packer build \
-var-file=../aws.json \
-var 'AwsProfile=itsandbox' \
-var 'AwsRegion=us-east-1' \
template.json
```

### CODE REVIEW

File: weechat/template.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      "secret_key": "{{user `AwsSecretAccessKey`}}",
+      "region": "{{user `Region`}}",
+      "vpc_id": "{{user `VpcId`}}",
+      "subnet_id": "{{user `VpcPublicSubnetId`}}",
+      "associate_public_ip_address": "true",
+      "security_group_id": "{{user `VpcPublicSecurityGroupId`}}",
+      "source_ami": "{{user `SourceImage`}}",
+      "instance_type": "{{user `InstanceType`}}",
+      "ssh_username": "{{user `SshUsername`}}",
+      "ssh_keypair_name" : "{{user `SshKeypairNam`}}",
```
Comment (tthyer) [inline weechat/template.json]: typo

**Comment by tthyer** [comment]
This is interesting and I generally approve fwiw since need to go learn more about packer.

File: weechat/template.json (hunk 1 above)
Comment (zaro0508) [reply weechat/template.json]: SshKeypairNam has been removed

**Comment by zaro0508** [comment]
I've replace this project with https://github.com/Sage-Bionetworks/ami-provisioner

**Pull request closed without merging**